docs/visualization/rgb.rst

```diff
@@ -60,21 +60,21 @@ it with Figure 1 of `Lupton et al. (2004)`_:
    :include-source:
```

Comment:
Unrelated, but while looking at the docs, I don't think the image above with noise is useful, given that there are other examples below with real images.